dotcom-rendering/src/components/Card/Card.tsx

```diff
@@ -491,6 +500,17 @@ export const Card = ({
 								/>
 							</TrailTextWrapper>
 						)}
+						{showLivePlayable && (
```

Comment:
I didn't immediately know what this means
Maybe showLatestLinks?

Reply:
That's the name as it comes from the data model. Do you think it's worth renaming at the enhancement step?

Reply:
oh weird, I guess less renaming of things the better? so maybe leave it alone :(

Reply:
That was my thinking, but agreed that the name could be clearer!